src/index.js

```diff
@@ -0,0 +1,111 @@
+import express from 'express';
+import { v4 as uuidv4 } from 'uuid';
+import _ from 'lodash';
+
+const PORT = Number(process.env.PORT) || 3000;
+const app = express();
```

Comment:
<img width="395" alt="image" src="https://user-images.githubusercontent.com/48351153/211278808-fc78bff8-c7d5-4b76-a493-8b7b0c266bbe.png">
`password` and `isDeleted` accessible through the API. I think these fields should never be returned to the user

Reply:
Removed `password` and `isDeleted` from responses.